### ISSUE TITLE
Creating a Wallet with Same Keys Creates Duplicate Wallet

### ISSUE THREAD

**Issue by dtrammell** [cryptoadvance/specter-desktop#1724]
When creating a wallet, if you define a wallet with the same device pool and same parameters (x of y multi-sig), Specter will create a duplicate of an existing wallet if a wallet with the same parameters had already been created previously.

**To Reproduce**
1. Create a wallet with some specific devices and parameters (2 of 3 for example)
2. Transfer a balance into the wallet
3. Create a second wallet with identical parameters
4. Note that the second wallet has the same balance and transaction history as the first wallet.

**Expected behavior**
I expected Specter to create the second wallet as a unique and different wallet.

**Desktop (please complete the following information):**
 - Where is your node running: local Bitcoin Core
 - Where is your specter running: local binary
 - If you're running a binary, which OS: Windows 11
 - Browser Chrome
 - Specter Version 1.9.2

**Additional context**
This issue can likely be fixed by allowing the user to increment the derivation paths of one or more device's keys so that the same set of keys is not used for the second wallet even though it is the same pool of signing devices.

**Comment by k9ert** [comment]
I don't understand your expected behaviour. We probably should prevent that a user is creating a second wallet like that but if he does, by definition that is the outcome because there is no difference in those wallets, there can't be any different outcome.

You could choose a different key within one of the wallets and then it would be a different wallet but if you choose the same key --> same wallet.

**Comment by k9ert** [comment]
In fact, this PR should prevent that you create the wallet twice:
https://github.com/cryptoadvance/specter-desktop/pull/1500

**Comment by k9ert** [comment]
I misunderstood my former PR: A new multisig wallet was checked for xpubs which are included in the new wallet twice. That's not what would help here. So with the new PR, the xpubs of a new wallet is now also checked against existing wallets.
Will close this issue when the PR is released.

**Comment by dtrammell** [comment]
Yes, I agree that Specter should detect that the user is attempting to create an exact copy of a wallet that already exists and warn against it.  Is it not possible to make the wallet unique even though it's using the same set of hardware devices by incrementing the derivation path on any one (or all) of the devices configured so that they're using different keys though?  This would allow the user to then create a unique wallet using the same set of hardware signing devices.

**Comment by dtrammell** [comment]
Creating a second unique wallet using the same set of hardware signing devices is the outcome I was attempting to achieve.

**Comment by k9ert** [comment]
Yes, definitely makes sense: You create a wallet and before that is settled, the system checks whether that already exists AND make a recommendation about using a different BIP44 account (probably for each of the wallets).
